scripts/aznfs_install.sh

```diff
@@ -138,7 +283,40 @@ verify_super_user()
     fi
 }
 
-if [ "$RELEASE_NUMBER" == "x.y.z" ]; then
+parse_user_config()
+{
+    if [ -f "$CONFIG_FILE" ]; then
+        # Read the value of AUTO_UPDATE_AZNFS from the configuration file.
+        AUTO_UPDATE_AZNFS=$(grep "^AUTO_UPDATE_AZNFS=" "$CONFIG_FILE" | cut -d '=' -f2)
```

Comment:
@linuxsmiths we are not handling the case if config file is present but is empty or AUTO_UPDATE_AZNFS is not present? 
@palashvij-msft we should handle this case as well.